vm_test.go

```diff
@@ -395,6 +395,16 @@ func TestFloatToValue(t *testing.T) {
 			true,
 			valueFloat(-0),
 		},
+		{
+			2.0000,
+			false,
```

Comment:
I believe we need it because of the test above due to the [changes](https://github.com/10gen/baas/commit/4de0fc691c4684e7c6153802dbadaa84065eb904) from a previous bug fix

Reply:
This test still passes without `expectedSign` value. What's it for? Maybe I miss anything.

Reply:
Ah I missed asserting the expected sign in the test 🤦 , good catch